src/test/resources/data/functional/controllers/accountEventController/updateAccountEvent.scenarios.json

```diff
@@ -0,0 +1,65 @@
+[
+    {
+      "description": "Should successfully update the event for given event id", 
+      "input": {
+        "body": {
+          "description":"Create Event",
+          "start_datetime": "2023-07-21T13:12:17.000+0000",
+          "end_datetime": "2023-07-22T13:12:17.000+0000"
+        },
+        "accountId": "0011e00000bWSxdAAG",
+        "eventId":"0691e000001X1yTAAS"
```

Comment:
Should there be test cases for start_datetime & end_datetime?

Reply:
Added